src/@providers/docker/services/deployment.ts

```diff
@@ -172,6 +154,49 @@ export class DockerDeploymentService extends CrudResourceService<'deployment', D
       throw new Error(stderr || 'Deployment failed');
     }
 
+    // Deployment successful. Report back to the server.
+    const inspectionRes = await this.inspect(containerName);
+    const networks = Object.keys(inspectionRes?.NetworkSettings.Networks || {});
+    const ipAddress = inspectionRes?.NetworkSettings.Networks[networks[0]].IPAddress;
+
+    for (const serviceConfig of inputs.services || []) {
+      const account = this.providerStore.getProvider(serviceConfig.account);
```

Comment:
This feels like eventually it will become a huge security risk. Allowing any provider access to all credentials creates a sort of honey pot situation.

Reply:
-- Had this comment on the wrong thread --

That's a weakness that already exists simply because the providerStore is injected into the providers/services. I agree with you, but let's discuss it outside the scope of this PR so we can move forward.

https://github.com/architect-team/arcctl/issues/110